R/tax_unique.R

```diff
@@ -0,0 +1,355 @@
+#' Filter occurrences to unique taxa
+#'
+#' A function to filter a list of taxonomic occurrences to unique taxa of a
+#' predefined resolution.
+#'
+#' @param occdf \code{dataframe}. A dataframe containing information on the
+#' occurrences or taxa to filter.
+#' @param binomial \code{character}. The name of the column in the dataframe
+#' containing the genus and species names of the occurrences, either in the
+#' form "genus species" or "genus_species".
+#' @param species \code{character}. The name of the column in the dataframe
```

Comment:
I agree that it would be nice, but given the flexibility in inputs, it seems easier to me to let people switch on the arguments they want, rather than switch off all the ones they don't. If you can think of a compromise on how to do this, let me know!

Reply:
Well, let's leave it as it is for now then! Ultimately, it works :-)